autonomy/cli/utils/click_utils.py

```diff
@@ -65,3 +66,22 @@ def wrapper(f: Callable) -> Callable:
         return f
 
     return wrapper
+
+
+def chain_selection_flag(
```

Comment:
What if my chain is not supported? There should be an easy way to pass in a set of full chain configs!

Reply:
Effectively a dict of service contract address and rpc

Reply:
I added flags `--rpc` and `--sca` (Service contract address) to facilitate this, now you can effectively pass the rpc and address as command line arguments.